source/_includes/custom/after_footer.html

```diff
@@ -1,3 +1,53 @@
 {% comment %}
-  Add content to be output at the bottom of each page. (You might use this for analytics scripts, for example)
 {% endcomment %}
+
+<script type="text/javascript">
+  // made header sticky
+  var sticky = new Waypoint.Sticky({
+    element: $("#banner"),
+    offset: -100
+  })
+
+  {% if page.opensource == true %}
```

Comment:
Having a little bit of trouble with code organization here - I think there must be a better way to give these methods their own JS files rather than just grouping them in the 'after footer' but I can't seem to get the backbone views to refer correctly to the DOM anywhere but running here.